pkg/monitor/datapath_drop.go

```diff
@@ -43,8 +43,8 @@ type DropNotify struct {
 
 // DumpInfo prints a summary of the drop messages.
 func (n *DropNotify) DumpInfo(data []byte) {
-	fmt.Printf("xx drop (%s) flow %#x to endpoint %d, identity %d->%d: %s\n",
-		api.DropReason(n.SubType), n.Hash, n.DstID, n.SrcLabel, n.DstLabel,
+	fmt.Printf("xx drop (%s) flow %#x to endpoint %d, identity %s->%s: %s\n",
+		api.DropReason(n.SubType), n.Hash, n.DstID, api.Entity(n.SrcLabel), api.Entity(n.DstLabel),
```

Comment:
I wonder if it would still be useful to also have something like a `-n`, `--numeric` option to still support printing the numeric identities in case we find it easier to read the numbers?

Reply:
I implemented this as the last commit.
@brb Does it also address your concern?